.github/workflows/nightly_publish.yml

```diff
@@ -77,6 +77,17 @@ jobs:
         with:
           name: mxnet-model-zoo
           path: mxnet/mxnet-model-zoo/build/reports
+      - name: Configure AWS Credentials
```

Comment:
@frankfliu Do we want to be publishing the updates to serving nightly and not as part of releases?

Reply:
I am thinking we should make it as a different job or even different workflow. The nightly publish is only tied to Maven uploads for now

Reply:
This should be part of release, instead of checking 'snapshot', it should be 'staging'

Reply:
Ok. I can change the input mode to staging.